page-components/featured-content-module.md

```diff
@@ -39,61 +39,45 @@ Featured content modules (FCMs) function to highlight a specific piece of conten
 
 <div class="content-67 content-first">
 
-Featured content modules should include a single piece of content from a defined content type, including: Blog, News, Video, Event, and Tool.
+Featured content modules should highlight a single piece of content. That content can be almost anything, from a webpage to a blog post to a PDF to a video.
 
 FCMs appear below the title and introductory paragraph on a page, and retain that position on all screen sizes.
 
 FCMs contain a headline, body copy text, and a graphical element related to the piece of content being featured. They are a “mini story” and never simply a dominant visual.
 
-FCMs can appear on any page type, though they are best suited for pages 3rd level and below in the site map. They are designed to accommodate pages with either a left-hand navigation or right-hand sidebar.
+FCMs can appear on any page type, though they are best suited for pages third level and below in the site map. They are designed to accommodate pages with either a left-hand navigation or right-hand sidebar.
 
 </div>
 
 <div class="content-33 content-last"></div>
 
 ## Style at desktop size
 
+{::nomarkdown}
 <div>
-![Image of featured content module example at desktop size]({{ site.baseurl }}/static/img/featured-content-module/fcm-desktop02.png)
+  <img src="{{ site.baseurl }}/static/img/featured-content-module/fcm-desktop02.png" alt="Image of featured content module example at desktop size">
 </div>
+{:/nomarkdown}
 
-<div class="content-33 content-first">
-* Module is a minimum of 320px total in height with 30px top and bottom padding. It appears 60px underneath the header/intro paragraph of the page.
-* Background color of module is Gray 5 with a 1px Gray 50 stroke.
-* Module bleeds to the edge of the page, right sidebar and left nav.
-* Visual spans 5 columns and bleeds into right sidebar. The visual is sized at 640 x 360px (a 16:9 ratio) for optimal appearance at all screen sizes.
-* Text spans 3 columns and is left and top aligned.
+<div class="content-50 content-first">
+* Module spans the full width of the page’s content area and is a minimum of 220px tall. It appears 60px underneath the header/intro paragraph of the page.
+* Background color of module is Gray 5 with a 1px Gray 40 stroke.
+* Visual is 270px wide and a minimum of 220px tall. The height of the visual will vary at different screen sizes so that the full height of the image is always visible. The aspect ratio of the image changes with screen size to keep the full image height visible; the image will be cropped up to 23% from each side.
+* Text spans 5 columns and is left and top aligned. There is 30px of padding around all sides of the text.
```

Comment:
Might want to do this.

```suggestion
* Text spans 5 columns (at maximum page width) and is left and top aligned. There is 30px of padding around all sides of the text. As available width dips below the maximum page width, the text area shrinks in width while the visual maintains a fixed width of 270px.
```

After writing that, I wonder if it'd be better take the responsive behavior mentions out of both above bullets and add a bullet dedicated to explaining how the layout shifts below maximum page width. Thoughts?

Reply:
I reworked this in eeb539d. Splitting out the responsive behavior into a separate bullet seemed like the way to go to me, and I think this section reads better now.